api/net/Areas/Subscriber/Controllers/WorkOrderController.cs

```diff
@@ -140,7 +140,7 @@ public async Task<IActionResult> RequestTranscriptionAsync(long contentId)
 
             // Send email to requestor to confirm we have receive their request for a transcript.
             var settingValue = _settingService.FindByName(_apiOptions.TranscriptRequestConfirmationKey)?.Value ?? "";
-            if (workOrder.RequestorId.HasValue && int.TryParse(settingValue, out int notificationId))
+            if (workOrder.RequestorId.HasValue && workOrder.Requestor != null && workOrder.Requestor.IsVacationMode != true && int.TryParse(settingValue, out int notificationId))
```

Comment:
A user can still request transcripts when on vacation.  The vacation mode is only about receiving emails.

Reply:
Fixed.